plugins/devtools/README.md

```diff
@@ -20,6 +20,20 @@ Lists the configuration being used by your current running Backstage instance.
 
 ![Example of Config tab](./docs/devtools-config-tab.png)
 
+#### Backstage Version Reporting
```

Comment:
The content and formatting of this is great, I just think it should be moved up to be under the `Info` heading as that's where the Backstage version is shown.

Reply:
Just below the image